pyproject.toml

```diff
@@ -1,64 +1,57 @@
-[tool.poetry]
+[project]
 name = "lacommunaute"
 version = "2.19.0"
 description = "La communauté de l'inclusion"
 authors = []
-license = "ETALAB 2.0"
+requires-python = "~=3.11"
 readme = "README.md"
+#license = "ETALAB 2.0"
+dependencies = [
+    "Django>=5.0,<5.1",
+    "python-dotenv>=0.21.0,<0.22",
+    "psycopg2-binary>=2.9.3,<3",
+    "django-machina>=1.2.0,<2",
+    "boto3>=1.24.91,<2",
+    "django-storages>=1.13.1,<2",
+    "httpx>=0.23.0,<0.24",
+    "django-compressor~=4.1",
+    "django-libsass>=0.9,<0.10",
+    "sentry-sdk~=2.8",
+    "django-social-share>=2.3.0,<3",
+    "django-htmx>=1.12.2,<2",
+    "django-taggit>=5.0.1,<6",
+    "django-csp~=3.7",
+    "django-permissions-policy>=4.15.0,<5",
+    "langdetect>=1.0.9,<2",
+    "freezegun>=1.5.1,<2",
+    "pyjwt>=2.9.0,<3",
+]
 
-[tool.poetry.dependencies]
-python = "^3.11"
-Django = "~5.0"
-python-dotenv = "^0.21.0"
-psycopg2-binary = "^2.9.3"
-django-machina = "^1.2.0"
-boto3 = "^1.24.91"
-django-storages = "^1.13.1"
-httpx = "^0.23.0"
-django-compressor = "^4.1"
-django-libsass = "^0.9"
-sentry-sdk = "^2.8"
-django-social-share = "^2.3.0"
-django-htmx = "^1.12.2"
-django-taggit = "^5.0.1"
-django-csp = "^3.7"
-django-permissions-policy = "^4.15.0"
-langdetect = "^1.0.9"
-freezegun = "^1.5.1"
-pyjwt = "^2.9.0"
-
-
-[tool.poetry.group.dev.dependencies]
-black = "^24.3.0"
-pylint = "^2.15.2"
-pylint-django = "^2.5.3"
-pre-commit = "^2.20.0"
-coverage = "^6.4.4"
-django-extensions = "^3.2.1"
-django-debug-toolbar = "^3.7.0"
-Werkzeug = "^3.0.3"
-pytest-django = "^4.5.2"
-debugpy = "^1.6.3"
-factory-boy = "^3.2.1"
-respx = "^0.20.0"
-pygithub = "^1.57"
-pytest-xdist = "^3.2.1"
-pillow = "^10.0.0"
-moto = "^4.2.2"
-virtualenv = "^20.24.6"
-ruff = "^0.4.5"
-djlint = "^1.34.1"
-syrupy = "^4.6.0"
-beautifulsoup4 = "^4.12.3"
-html5lib = "^1.1"
-
-[build-system]
-requires = ["poetry-core"]
-build-backend = "poetry.core.masonry.api"
-
-[tool.poe.tasks]
-export = { shell = "poetry export --output requirements/base.txt" }
-export_dev = { shell = "poetry export --with dev --output requirements/dev.txt" }
+[dependency-groups]
+dev = [
+    "black>=24.3.0,<25",
+    "pylint>=2.15.2,<3",
+    "pylint-django>=2.5.3,<3",
+    "pre-commit>=2.20.0,<3",
+    "coverage>=6.4.4,<7",
+    "django-extensions>=3.2.1,<4",
+    "django-debug-toolbar>=3.7.0,<4",
+    "Werkzeug>=3.0.3,<4",
+    "pytest-django>=4.5.2,<5",
+    "debugpy>=1.6.3,<2",
+    "factory-boy>=3.2.1,<4",
+    "respx>=0.20.0,<0.21",
+    "pygithub~=1.57",
+    "pytest-xdist>=3.2.1,<4",
+    "pillow>=10.0.0,<11",
+    "moto>=4.2.2,<5",
+    "virtualenv>=20.24.6,<21",
+    "ruff>=0.4.5,<0.5",
+    "djlint>=1.34.1,<2",
+    "syrupy>=4.6.0,<5",
+    "beautifulsoup4>=4.12.3,<5",
+    "html5lib~=1.1",
```

Comment:
c'est marrant que tu sois passé d'un coup aux quantifiers avec des < ; quelle en est la raison ?

Reply:
pas de bonne raison, c'est le rendu de `uvx migrate-to-uv`. à creuser …